fastly/dictionary_item.go

```diff
@@ -189,6 +189,49 @@ func (c *Client) UpdateDictionaryItem(i *UpdateDictionaryItemInput) (*Dictionary
 	return b, nil
 }
 
+type BatchModifyDictionaryItemsInput struct {
+	Service    string `json:"-,"`
+	Dictionary string `json:"-,"`
+
+	Items []BatchDictionaryItem `json:"items"`
+}
+
+type BatchDictionaryItem struct {
+
+
+	Operation BatchOperation `json:"op"`
+	ItemKey   string         `json:"item_key"`
+	ItemValue string         `json:"item_value"`
```

Comment:
Why not using github.com/ajg/form as we usually do for input structs?

Reply:
I used the json encoding/decoding because the documentation for the end point showed that the content type was application/json and not application/x-www-form-urlencoded.